### PR TITLE
[stable/kong] Add a chart for Kong

### PR DESCRIPTION
**Pull request by shashiranjan84**
- Chart deploys Kong and its backing datastore either Cassandra or
  PostgreSql
- Uses official [Kong](https://hub.docker.com/_/kong/) Docker image

### CODE REVIEW

**Comment by shashiranjan84** [comment]
Hi @prydonius, would you please review this PR?

**Comment by shashiranjan84** [comment]
Not sure why it removed `cncf-cla: yes` tag for just putting a comment.

image

File: stable/kong/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+spec:
+  replicas: {{ .Values.replicaCount }}
+  template:
+    metadata:
+    {{- if .Values.podAnnotations }}
+      annotations:
+{{ toYaml .Values.podAnnotations | indent 8 }}
+    {{- end }}
+      labels:
+        app: "{{ template "kong.fullname" . }}"
```
Comment (unguiculus) [inline stable/kong/templates/deployment.yaml]: Please make sure you read our review guidelines and best practices.

Please change throughout the chart: 
`app: "{{ template "kong.name" . }}"`

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: stable/kong/templates/deployment.yaml (hunk 1 above)
Comment (shashiranjan84) [reply stable/kong/templates/deployment.yaml]: Sure @unguiculus, I'll update them. But I did notice `"{{ template "<chart>.fullname" . }}` being used in many of the charts.

Comment (unguiculus) [reply stable/kong/templates/deployment.yaml]: Current best practice is to use the name template.

Comment (shashiranjan84) [reply stable/kong/templates/deployment.yaml]: Thanks, I am in process of updating it.

**Comment by shashiranjan84** [comment]
Closing PR till I finish all changes.

**Pull request closed without merging**